comp/otelcol/collector/impl/collector.go

```diff
@@ -196,6 +198,8 @@ func NewComponentNoAgent(reqs RequiresNoAgent) (Provides, error) {
 	}
 	factories.Connectors[component.MustNewType("datadog")] = datadogconnector.NewFactory()
 	factories.Extensions[ddextension.Type] = ddextension.NewFactoryForAgent(&factories, newConfigProviderSettings(reqs.URIs, reqs.Converter, false))
+	// Not sure what to do here. The extension requires the trace-agent, so likely we shouldn't add it to the factories at all.
+	// factories.Extensions[ddprofilingextension.Type] = ddprofilingextension.NewFactory()
```

Comment:
remove this .  Once we support extension with OCB, we can it here

Reply:
I've removed this. 

> Once we support extension with OCB

The extension is already supported with OCB. See PR description `How to run the extension locally (OSS Collector)`. It's just that we agreed no direct intake in OTel agent, so here I can't use that path.